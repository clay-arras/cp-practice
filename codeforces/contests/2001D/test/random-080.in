1
14
7 21 9 1 23 8 8 11 18 24 3 1 23 5
