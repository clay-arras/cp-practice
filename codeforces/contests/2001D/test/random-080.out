11
7 21 9 1 23 8 11 18 24 3 5
